src/main/java/com/faforever/client/vault/review/ReviewController.java

```diff
@@ -76,7 +77,7 @@ public void initialize() {
   public void setReview(Optional<Review> optionalReview) {
```

Comment:
I know you didn't write this code, but I like to have mistakes fixed when I see them.

This method is the perfect example why you only ever return Optionals but never use them as parameters:

- The code checks for the Optional. If it's missing, you return. (The orElse feature is not even used)
- Due to that the code forgets to check if the Optional is null (this is always possible)
- The optional needs to be unwrapped.

Instead of checking for Optional it would be more useful to check for null instead. If the caller has an Optional hat hand he can just do .orElse(null)